app/src/main/java/chat/rocket/android/service/ServerConnectivity.java

```diff
@@ -16,4 +16,10 @@ public ServerConnectivity(String hostname, int state) {
     this.hostname = hostname;
     this.state = state;
   }
+
+  public static class Disconnected extends Exception {
```

Comment:
Exception is something really strong. What about naming this class `DisconnectedException` ?